addons/account_peppol/views/account_move_views.xml

```diff
@@ -53,13 +52,14 @@
         </field>
     </record>
 
+    <!-- to be removed in master -->
     <record id="account_peppol_view_in_invoice_bill_tree_inherit" model="ir.ui.view">
         <field name="name">account.out.invoice.tree.inherit</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_in_invoice_tree"/>
         <field name="arch" type="xml">
             <field name="state" position="before">
-                <field name="peppol_move_state" optional="hide"/>
+                <field name="peppol_move_state" invisible="1"/>
```

Comment:
why this diff if you need to remove it in master?

Reply:
because we can't remove this record in stable but the field needs to be hidden